frontends/election-manager/src/lib/backends/admin_backend.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  ElectionDefinition,
+  Id,
+  IdSchema,
+  safeParse,
+  safeParseElectionDefinition,
+} from '@votingworks/types';
+import { fetchJson } from '@votingworks/utils';
+import { Admin } from '@votingworks/api';
+import { ElectionManagerStoreStorageBackend } from './storage_backend';
+
+/** @visibleForTesting */
+export const activeElectionIdStorageKey = 'activeElectionId';
+
+export class ElectionManagerStoreAdminBackend extends ElectionManagerStoreStorageBackend {
+  private async loadActiveElectionId(): Promise<Id | undefined> {
```

Comment:
I'm a bit confused - why is there some local storage state in this backend? I expected the backend to just be a stateless interface for making fetch requests.

Reply:
I modeled it as the client storing the current election that it's working with, but (long term) that's it. I toyed with having a `active` flag on `elections`, but decided it was better to just have the client manage it. In the future we'll probably want to present UI to select and switch elections, and this approach doesn't encode any server-side assumptions about that.

Reply:
Assuming the computer's clock is correct, we could use the election date in the election definition to determine which elections are in the past.

Reply:
> Assuming the computer's clock is correct, we could use the election date in the election definition to determine which elections are in the past.

True, but there might be more than one in the future. Especially if you have one that's a "sandbox" or something.

Reply:
Ah interesting! I can totally see the need for having UI state for "this is the election I'm currently working with" and wanting that state to be remembered across restarts.

I don't think this will be the only piece of UI state with these characteristics, so it's worth thinking about a pattern we can use a bit more broadly.

I think local storage is a very sensible place to put that state, even though there's no real difference since we have a single-client/single-server architecture. It seems nice to keep the UI state/user preferences separate from the app domain state.

Maybe it would be clearer to not have this managed in the backend adapter but instead in the app itself so that we can easily reason about it and add future UI state to it as well.